vocabularies/UI.xml

```diff
@@ -1446,6 +1446,10 @@ It is NOT tied to a data value (in contrast to [DataFieldWithIntentBasedNavigati
         <Property Name="Action" Type="Edm.String" Nullable="true">
           <Annotation Term="Core.Description" String="Name of the Action on the Semantic Object. If not specified, let user choose which of the available actions to trigger." />
         </Property>
+        <Property Name="NavigationAvailable" Type="Edm.Boolean" Nullable="true" DefaultValue="true">

```

Comment:
```suggestion
        <Property Name="NavigationAvailable" Type="Edm.Boolean" Nullable="false" DefaultValue="true">
```

I assume we do not want `null` here

Reply:
Hmm, yes you are right. I thought at first it doesn't matter, but I agree now ...